cmd/substreams-sink-postgres/generate_csv.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/streamingfast/cli"
+	. "github.com/streamingfast/cli"
+	"github.com/streamingfast/cli/sflags"
+	"github.com/streamingfast/shutter"
+	sink "github.com/streamingfast/substreams-sink"
+	"github.com/streamingfast/substreams-sink-postgres/db"
+	"github.com/streamingfast/substreams-sink-postgres/sinker"
+	"go.uber.org/zap"
+)
+
+var generateCsvCmd = Command(generateCsvE,
+	"generate-csv <psql_dsn> <endpoint> <manifest> <module> [<start>:<stop>]",
+	"Runs first load sink process",
+	ExactArgs(6),
+	Flags(func(flags *pflag.FlagSet) {
+		sink.AddFlagsToSet(flags)
```

Comment:
The `generate-csv` sinker must run against final blocks only IMO, I don't see good reason to deal with "live" blocks. Let's remove the flag here (I left another command to force the stream to be on final blocks only).
```suggestion
		sink.AddFlagsToSet(flags, sink.FlagIgnore("final-blocks-only"))
```

Reply:
Ok, done!